packages/generator-joplin/generators/app/templates/webpack.config.js

```diff
@@ -93,19 +93,30 @@ function validateCategories(categories) {
 
 function validateScreenshots(screenshots) {
 	if (!screenshots) return null;
-	// eslint-disable-next-line github/array-foreach -- Old code before rule was applied
-	screenshots.forEach(screenshot => {
+	for (const screenshot of screenshots) {
 		if (!screenshot.src) throw new Error('You must specify a src for each screenshot');
 
+		// Avoid attempting to download and verify URL screenshots.
+		if (screenshot.src.startsWith('https://') || screenshot.src.startsWith('http://')) {
+			continue;
+		}
+
 		const screenshotType = screenshot.src.split('.').pop();
 		if (!allPossibleScreenshotsType.includes(screenshotType)) throw new Error(`${screenshotType} is not a valid screenshot type. Valid types are: \n${allPossibleScreenshotsType}\n`);
 
-		const screenshotPath = path.resolve(srcDir, screenshot.src);
+		let screenshotPath;
+		if (screenshot.src.startsWith('./') || screenshot.src.startsWith('../')) {
+			// If screenshot srcs start with ./ or ../, they're relative to the src directory.
+			screenshotPath = path.resolve(srcDir, screenshot.src);
+		} else {
+			screenshotPath = path.resolve(rootDir, screenshot.src);
+		}
+
 		// Max file size is 1MB
 		const fileMaxSize = 1024;
 		const fileSize = fs.statSync(screenshotPath).size / 1024;
 		if (fileSize > fileMaxSize) throw new Error(`Max screenshot file size is ${fileMaxSize}KB. ${screenshotPath} is ${fileSize}KB`);
```

Comment:
Since we aren't hosting the screenshots, this may no longer be relevant.

Reply:
I guess this is good to leave this to prevent very large screenshots to be accidentally used.